app/locale/ca.coffee

```diff
@@ -77,7 +164,7 @@ module.exports = nativeDescription: "Català", englishDescription: "Catalan", tr
     contribute: "Col·laborar"
     legal: "Legalitat"
     privacy: "Privadesa"
-    about: "Sobre Nosaltres"
+    about: "Sobre Nosaltres" # {change}
```

Comment:
Confirm this should have changed - it hasn't changed in other files

Reply:
Also changed in `de-de`

Reply:
It should not have changed -  the issue was due to duplicate 'about' tag names in different categories.  Regex did not account for this.  Fixed.